RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java

```diff
@@ -174,8 +180,12 @@ public void process(Content dataSource, IngestJobContext context, DataSourceInge
     private void getHistory(String browser, String browserLocation) {
         FileManager fileManager = currentCase.getServices().getFileManager();
         List<AbstractFile> historyFiles;
+        String historyFileName = HISTORY_FILE_NAME;
+        if (browser == NbBundle.getMessage(Chromium.class, "Browser.name.UC.Browser")) {
```

Comment:
Should this be .equals() instead? There are several of these.

Reply:
Changed to .equals() where needed.